src/OpenTelemetry.Exporter.GoogleCloud/OpenTelemetry.Exporter.GoogleCloud.csproj

```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="[3.8.0,4.0)" />
-    <PackageReference Include="Google.Cloud.Trace.V2" Version="[3.5.0,4.0)" />
+    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="3.15.0" />
+    <PackageReference Include="Google.Cloud.Trace.V2" Version="3.6.0" />
```

Comment:
Are you sure that this package will work with 4.0+ verion?
If not, please bring bang older format `[xxxx,4.0)`